### ISSUE TITLE
Apps on different Users not shown in AM Profiles

### ISSUE THREAD

**Issue by BurhanDanger** [MuntashirAkon/AppManager#643]
- [x] I know what my device, OS and App Manager versions are
- [ ] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
Apps installed on different users doesn't show in profiles selection. Also `Add to profile` does not work reliably. 

**Device info**
 - Device: Curtana
 - OS Version: A11
 - App Manager Version: v2.6.5.1
 - Mode: root

Related https://github.com/MuntashirAkon/AppManager/issues/626

**Comment by MuntashirAkon** [comment]
Are you sure? It works on Lineage OS 18.1 (Android 11).

**Comment by BurhanDanger** [comment]
It could be due to rom. I'm not sure what kind of logs would be useful. 

![Screenshot_20211204-131941539 (1)](https://user-images.githubusercontent.com/25033866/144701491-8644af09-41ea-48aa-905c-875c50a3aa67.jpg)
![Screenshot_20211204-132036307 (1)](https://user-images.githubusercontent.com/25033866/144701492-797fb576-4d64-4581-9de1-de8a053711e3.jpg)

I might try that in a week after clean flashing ROM. Closing this now

**Comment by BurhanDanger** [comment]
Problem persists on official LineageOS. 

https://user-images.githubusercontent.com/25033866/145869045-3f43024e-361f-4d05-b56c-6ebfe4695aee.mp4

Logcat:
[AM_Log.txt](https://github.com/MuntashirAkon/AppManager/files/7706084/AM_Log.txt)

**Comment by MuntashirAkon** [comment]
Apps installed for other users still show up on my device both debug and stable. Just to check if this is an error, I've created a new user and installed a brand new app there which still showed up in the profiles section. So, it might not be an issue with users at all.

Did you check if these package names show up in the output of
```sh
pm list packages -u --user 0
```
?

**Comment by BurhanDanger** [comment]
> Did you check if these package names show up in the output of
> 
> ```shell
> pm list packages -u --user 0
> ```
> 
> ?

Packages don't show up there. Why would they ? These packages are not installed on user 0. 

Consider following case: 

```
curtana:/ $ pm install --user 10 /data/local/tmp/floris.apk
Success
curtana:/ $ pm list packages -u --user 0 | grep -i floris
1|curtana:/ $ pm list packages -u --user 10 | grep -i floris
package:dev.patrickgold.florisboard

```

**Comment by MuntashirAkon** [comment]
Fixed in 6d453c8ddfbad2791e96ba7a7925abf9763fe63a
